.env.local.example

```diff
@@ -3,4 +3,5 @@ NEXT_PUBLIC_CAPTCHA_KEY=
 
 NEXT_PUBLIC_MATOMO_URL_BASE=https://analytics.hel.ninja/
 NEXT_PUBLIC_MATOMO_SITE_ID=68
-NEXT_PUBLIC_MATOMO_ENABLED=false
\ No newline at end of file
+NEXT_PUBLIC_MATOMO_ENABLED=false
+NEXT_PUBLIC_LANGUAGE_CIMODE_VISIBLE=false
```

Comment:
Should be added to Dockerfile ARG also

Reply:
https://github.com/City-of-Helsinki/palvelutarjotin-ui/pull/120